automation/Dockerfile.fedora

```diff
@@ -6,7 +6,12 @@ FROM fedora
 ENV container docker
 COPY docker_enable_systemd.sh docker_sys_config.sh ./
 
-RUN bash ./docker_enable_systemd.sh && rm ./docker_enable_systemd.sh
+RUN \
+    dnf -y update && \
+    dnf install -y systemd-udev && \
```

Comment:
IMHO this should be added to the NM section in the other dnf command. Also a reference to https://bugzilla.redhat.com/show_bug.cgi?id=1704456 would be great so it is clear that it can be removed eventually.

Reply:
I somehow considered the udev, part of systemd patching, that is why I placed it here.
But I guess it is not related, I will move it.